validator/src/lib.rs

```diff
@@ -72,17 +55,19 @@ fn download_tar_bz2(
     let progress_bar = new_spinner_progress_bar();
     progress_bar.set_message(&format!("{}Downloading {}...", TRUCK, url));
 
-    let client = reqwest::Client::new();
-    let response = client
-        .get(url.as_str())
-        .send()
-        .and_then(|response| response.error_for_status())
-        .map_err(|err| format!("Unable to get: {:?}", err))?;
+    let client = ureq::agent();
```

Comment:
It looks like you're working off an older version of the code instead of the HEAD of the master branch.  We switched from ureq to reqwest maybe a week ago now, so all these changes shouldn't be present.

Reply:
it seems i just have to copy validator/src/lib.rs of master branch to my branch right?

Reply:
I'd get the latest lib.rs from the master branch, then only add in your change (which if I recall is just a single line)